ui/src/templates/api/index.ts

```diff
@@ -52,9 +55,17 @@ export const createDashboardFromTemplate = async (
   return dashboard
 }
 
-const createLabelsFromTemplate = async (
+const createDashboardLabelsFromTemplate = async (
   template: DashboardTemplate,
   dashboard: IDashboard
+) => {
+  const templateLabels = await createLabelsFromTemplate(template, dashboard)
+  await client.dashboards.addLabels(dashboard.id, templateLabels)
+}
+
+const createLabelsFromTemplate = async <T extends TemplateBase, R>(
+  template: T,
+  resource: R & {orgID: string}
```

Comment:
perhaps this should just accept orgID directly instead of a resource with an orgID